### PR TITLE
fix: correctly import declarations for member functions

### PR DESCRIPTION
**Pull request by WinPlay02**
- fix: correctly import declarations for member functions
- replace import functionality

### CODE REVIEW

File: packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts [hunk 1]
```diff
@@ -1020,7 +1020,12 @@ export class SafeDsPythonGenerator {
         );
         const fullyQualifiedTargetName = this.generateFullyQualifiedFunctionName(expression);
         if (!containsOptionalArgs && isSdsMemberAccess(expression.receiver)) {
-            const referenceImport = this.createImportDataForReference(expression.receiver.member!);
+            const classDeclaration = this.getOutermostContainerOfType(callable, isSdsClass)!;
+            const referenceImport = this.createImportDataForNode(
+                classDeclaration,
+                expression.receiver.member!,
+                classDeclaration.name,
```
Comment (WinPlay02) [inline packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts:1027]: I'm still a bit doubtful about the `refName` (3rd parameter) here, but considering that the user has no way of specifying it here, that should be fine.

Comment (lars-reimann) [reply packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts:1027]: Maybe we can make the 3rd parameter optional, with `declaration.name` as the default?

**Review by lars-reimann — APPROVED** [review]
Looks good. Code like

```
package test

pipeline myPipeline {
    val titanic = Table.fromCsvFile("C:/Users/Lars/OneDrive/Desktop/test/titanic.csv");
    val age = titanic.getColumn("age");
    val idness = age.idness();
    val string = idness.toString();
}
```

can now be executed.

**Pull request merged**